templates/vue/src/components/TapestryApp.vue

```diff
@@ -28,9 +28,10 @@ export default {
     return {}
   },
   computed: {
-    ...mapState(["nodes", "links", "selection", "settings", "rootId"]),
+    ...mapState(["nodes", "links", "selection", "settings"]),
+    ...mapGetters(["isEmptyTapestry"]),
     isSidebarOpen() {
-      return Boolean(this.$route.query.sidebar)
+      return Boolean(this.$route.query.sidebar) && !this.isEmptyTapestry
```

Comment:
Just wondering, what is the purpose of adding the condition that the Tapestry not be empty here?

Reply:
On an empty Tapestry, the sidebar is hidden but the main tapestry view still leaves space for the sidebar, resulting in an awkward white space on the right. Adding this just makes the spacing consistent with the actual visibility of the sidebar.

Reply:
Makes sense now. Thanks for explaining.